app/scripts/views/form.js

```diff
@@ -271,6 +271,26 @@ function (_, $, p, Validate, AuthErrors, BaseView, Tooltip,
      * or showValidationErrorsEnd to display view specific messages.
      */
     showValidationErrors: function () {
+      function showFirstError (inputEls) {
```

Comment:
I like this cleanup, the function name makes the intent of the code much clearer.